### ISSUE TITLE
Zig Zen is outdated

### ISSUE THREAD

**Issue by EleanorNB** [fengb/zigbot9001#24]
The Zen has been updated since the `%%`zen` command was added. The new version reads:
```
* Communicate intent precisely.
* Edge cases matter.
* Favor reading code over writing code.
* Only one obvious way to do things.
* Runtime crashes are better than bugs.
* Compile errors are better than runtime crashes.
* Incremental improvements.
* Avoid local maximums.
* Reduce the amount one must remember.
* Focus on code rather than style.
* Resource allocation may fail; resource deallocation must succeed.
* Memory is a resource.
* Together we serve the users.
```
Apologies to throw this over the fence; it's 3:30AM over here, I'll make a PR when I wake up.

**Comment by EleanorNB** [comment]
Closed in #25.

**Comment by fengb** [comment]
Thanks!
